tests/SprykerSdkTest/Spryk/Integration/Glue/Processor/Mapper/AddGlueResourceMapperMethodTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Spryk\Integration\Glue\Processor\Mapper;
+
+use Codeception\Test\Unit;
+use SprykerSdkTest\Module\ClassName;
+
+/**
+ * Auto-generated group annotations
+ * @group SprykerSdkTest
+ * @group Spryk
+ * @group Integration
+ * @group Glue
+ * @group Processor
+ * @group AddGlueProcessorModelInterfaceTest
+ * Add your own group annotations below this line
+ */
+class AddGlueResourceMapperMethodTest extends Unit
+{
+    /**
+     * @var \SprykerSdkTest\SprykIntegrationTester
+     */
+    protected $tester;
+
+    /**
+     * @return void
+     */
+    public function testAddsGlueResourceMapperMethod(): void
+    {
+        $this->tester->run($this, [
+            '--module' => 'FooBar',
+            '--resourceType' => 'foo-bars',
+            '--toTransfer' => 'BarQuxTransfer',
```

Comment:
I understand it's just the same, but maybe it's worth to make another pair of tests like these just to cover the case when the `toTransfer` is default and the `fromTransfer` is passed. 

And then another pair for both being set. 

What do you think?